extension/tsconfig.json

```diff
@@ -20,10 +20,11 @@
         /* Strict Type-Checking Option */
         "strict": true, /* enable all strict type-checking options */
         /* Additional Checks */
-        "noUnusedLocals": true, /* Report errors on unused locals. */
+        "noUnusedLocals": false, /* Report errors on unused locals. */
```

Comment:
did this change really help?

Reply:
Yes, it did what I wanted. But that should be reverted. It doesn't feel right to change the default tsconfig to attain the behavior I was looking for in GitHub Workflow. Even though ESLint picks up the slack. There's probably a good way to do this but I'll save that trouble for another time :)

Reply:
Reverted.